### PR TITLE
Cimpl doc mk

### PR DESCRIPTION
**Pull request by markkramerus**
Addressed comments from last PR. Also changed a few capitalization issues for consistency.

### CODE REVIEW

File: docs/cimpl6ObjectiveFHIR.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ _This is an introductory guide to CIMPL's class library: **Objective FHIR Versio
 
 Objective FHIR ("OBF") is an [**object-oriented**](https://en.wikipedia.org/wiki/Object-oriented_programming) abstraction of FHIR. It provides modelers a way to define a detailed clinical information model by subclassing, extending, and constraining a pre-existing class library. These classes can then be translated automatically into FHIR profiles, FHIR Implementation Guides, data dictionaries, schemas, and other assets, using a choice of the three major FHIR versions: DSTU 2, STU 3, and R4.
 
-OBF serves as the base class library for the Clinical Information Modeling and Profiling Language (CIMPL). Use of OBF with CIMPL is optional, but recommended. _Tabula rasa_ (blank slate) use of CIMPL is also possible and appropriate for some projects. For more information on the choice between using OBF and the _tabula rasa_ approach, see [CIMPL 6 User Guide](cimpl6UserGuide.md).
+OBF serves as the base class library for the Clinical Information Modeling and Profiling Language (CIMPL). Use of OBF with CIMPL is optional, but recommended. "blank slate" use of CIMPL is also possible and appropriate for some projects. 
+
+(**TO DO: Move discussion of Blank Slate approach to here**)
```
Comment (mlterryMitre) [inline docs/cimpl6ObjectiveFHIR.md]: lmk if we want to keep this **TO DO:** note prior to publishing.

Comment (markkramerus) [reply docs/cimpl6ObjectiveFHIR.md]: we can keep. I'm not shy about "always in beta"

Comment (markkramerus) [reply docs/cimpl6ObjectiveFHIR.md]: ... and there are several other "TO DO"s

**Comment by markkramerus** [comment]
Revised mkdocs.yml.  Merging....

**Pull request merged**